app/src/main/res/layout/ride_history_cardview.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginHorizontal="24dp"
```

Comment:
Extract raw dimens.
Also, instead of wrapping the CL in a CardView, you can use a background drawable for the CL to provide it a shape. This way you reduce nesting and the number of inflated views

Reply:
changed it in the meantime